SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1168,15 +1168,17 @@ void USpatialReceiver::HandleRPC(const Worker_ComponentUpdateOp& Op)
 		}
 	}
 
-	Schema_Object* EventsObject = Schema_GetComponentUpdateEvents(Op.update.schema_type);
-
-	Schema_FieldId EventId = SpatialConstants::UNREAL_RPC_ENDPOINT_EVENT_ID;
-	// Client and Server Unreliable RPCs can be packed, in which case they're sent using a different event ID.
-	if (GetDefault<USpatialGDKSettings>()->bPackRPCs && Op.update.component_id != SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID)
+	ProcessRPCEventField(EntityId, Op, RPCEndpointComponentId, false);
```

Comment:
Could you add a comment about what's going on here

Reply:
Done